src/components/QueryBuilder.vue

```diff
@@ -119,6 +119,10 @@ export default Vue.extend( {
 				document
 					.getElementsByClassName( 'querybuilder__condition-wrapper-last' )[ 0 ]
 					.scrollIntoView( { behavior: 'smooth' } );
+
+				const toggle = document.getElementsByClassName( 'querybuilder__condition-relation-toggle' );
+				( toggle[ toggle.length - 1 ] as HTMLElement ).setAttribute( 'tabindex', '0' );
```

Comment:
@micgro42 I think this line can/ should be Implemented in wikit ?

Reply:
Why do you want to set the focus of the toggle group as a whole instead of one of its buttons (probably the not selected one)?

Reply:
you're right. It doesn't make sense to put the focus on the whole group. 
I change it to focus on the non-selected button